cmake/targets/ara/apps/common/runtime.mk

```diff
@@ -0,0 +1,142 @@
+# Copyright 2020 ETH Zurich and University of Bologna.
```

Comment:
file can be removed. it shouldn't be required anymore since you ported it to cmake.